boilerplate/README.md

```diff
@@ -0,0 +1,5 @@
+# Zapier Platform Boilerplate
+
+Boilerplate is an empty Zapier integration project. We use it to build a zip
+providing all the necessary dependencies for integrations developed on visual
+builder.
```

Comment:
The `boilerplate` has been moved from the `REPO_ROOT/packages/core/boilerplate` to its own directory `REPO_ROOT/boilerplate`. Putting `boilerplate` in `core` is awkward. It forces `core` to aware of the existence of `legacy-scripting-runner` because `boilerplate` depends on both `core` and `legacy-scripting-runner`.  It makes more sense for `boilerplate` to be isolated.

I thought about putting it to `REPO_ROOT/packages/boilerplate`. It wasn't an idea because `boilerplate` is not a yarn workspace. We don't want yarn to hoist dependencies to workspace root's `node_modules` when building the boilerplate zip.